app/controllers/games_controller.rb

```diff
@@ -0,0 +1,34 @@
+class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def index
+  end
+
+  def create
+    @game = Game.create(game_params)
+    @game.update_attribute(white_player_id, current_user.id)
+    redirect_to root_path
+  end
+
+  def show 
+    @game = Game.find_by_id(params[:id])
+    if @game.blank?
+      render text: 'Not Found', status: :not_found
+    end
+  end
+
+  def update 
+    @game = Game.find(params[:id])
+    @game.update_attributes(black_player_id: current_user.id)
+    redirect_to game_path(@game.id)
+  end
```

Comment:
We wouldn't want to do any sort of logic like that in the controller, that would be where the model comes in. 

Choice of color isn't an explicit requirement of the game though, so maybe a ticket & subsequent team vote to add it? Or has this already been decided via wireframes/other convos, and I'm just forgetting things?